docs/source/examples/mitiq-paper/mitiq-paper-codeblocks.ipynb

```diff
@@ -0,0 +1,1087 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "wjnIQWX8d5UY"
+   },
+   "source": [
+    "# Mitiq paper codeblocks\n",
+    "\n",
+    "Codeblocks from the main text of the [Mitiq paper](https://arxiv.org/abs/2009.04417) [1]."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "DRplhovffVfk"
+   },
+   "source": [
+    "**Codeblock 1**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "YTOaB_3tdymW",
+    "outputId": "38c2ce43-463f-4ab8-e402-fbc1d5877ce3"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[K     |████████████████████████████████| 122kB 3.7MB/s \n",
```

Comment:
I take it there is no way to really suppress these dialogs here?

Reply:
I don't know.

Reply:
It's from the pip install on colab. You can just delete the output cell.